jaxrs-tck/src/main/java/jakarta/ws/rs/tck/basic/Util.java

```diff
@@ -0,0 +1,20 @@
+/*******************************************************************
+* Copyright (c) 2021 Eclipse Foundation
+*
+* This specification document is made available under the terms
+* of the Eclipse Foundation Specification License v1.0, which is
+* available at https://www.eclipse.org/legal/efsl.php.
+*******************************************************************/
+package jakarta.ws.rs.tck.basic;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
+public class Util {
+    public static final int HTTP_PORT = Integer.getInteger("http.port", 8010);
```

Comment:
Sorry for the confusion on this - that port should be set by the vendor to tell the test framework what port its server is listening on.  Since the server port is controlled outside of the TCK framework (including Arquillian, etc.) it is the responsibility of the vendor to tell the framework what port it is using.  I was using port 8010 which is why I used that as the default.  I could change that be a more widely-used default port (like 8080), but defaulting to zero won't work.

Reply:
Isn't there a way the TCK could force a specific port to be used by the vendor's server?

The Java SE Bootstrap is really beneficial here. Too bad that we don't force all servers to support it.

Reply:
Using 8080 as default may be a better option, but otherwise LGTM.